docs/pages/config-reference.mdx

```diff
@@ -4,24 +4,43 @@ sidebar_label: Config Reference
 ---
 
 ### Syncer Flags
+Before using any particular flag mentioned below, we recommend making yourself familiar with the documentation pages that are related to the topic addressed by the flag and default to using the flags and helm variables as described in the documentation. 
```

Comment:
Here I am trying to say that the user should read and follow the docs, and preferably use documented helm values, rather than just scrolling through the flags list and just picking from here. I am unsure if that comes across clearly enough in the sentence I've written.